### PR TITLE
Create comment form for popup modal.

### PR DESCRIPTION
**Pull request by oyhpnayiaw**
In this pull request. I refactor some of the previous code and create form for popup modal. 

Changes.
- Move all of API related codes into separate file.
- Add comment form!!. When the user clicks on the "Comment" button, the data is recorded in the [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) and the screen is updated.

### CODE REVIEW

File: src/modules/api.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import { API_URL, INVOLVEMENT_API_URL, appID } from '../config.js';
```
Comment (Timbar09) [inline src/modules/api.js:1]: Creating this file is a great idea, @oyhpnayiaw. I'll also move my API-related code into this file.

File: src/modules/popup.js [hunk 2]
```diff
@@ -72,14 +53,59 @@ const openPopup = async (detail, popupEl, bodyEl = document.body) => {
...
+    // insert the comment after comment title to make latest comment showing on top
+    popupEl
+      .querySelector('.popup-comment-section .popup-title')
+      .insertAdjacentElement('afterend', commentEl);
+
+    commentCountEl.textContent = Number(commentCountEl.textContent) + 1;
+
+    button.disabled = false;
+    button.innerHTML = 'Comment';
+    e.target.reset();
```
Comment (Timbar09) [inline src/modules/popup.js:106]: The form code looks good, bro.

File: src/modules/api.js (hunk 1 above)
Comment (oyhpnayiaw) [reply src/modules/api.js:1]: Good man, our codebase will become more manageable and readable 💪

**Review by Timbar09 — APPROVED** [review]
# **The form code is good.** ✔️ 

It's safe to merge. ✅ 
There is a couple of appreciation comments above.

Good job as always, bro. 👍

**Pull request merged**